packages/odyssey-react-mui/src/theme/mixins.ts

```diff
@@ -10,12 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import type { ThemeOptions } from "@mui/material";
+import type { MixinsOptions } from "@mui/material";
 import * as Tokens from "@okta/odyssey-design-tokens";
+import { TokenOverrideOptions } from ".";
+import { deepmerge } from "@mui/utils";
 
-export const mixins: ThemeOptions["mixins"] = {
-  maxWidth: Tokens.FontLineLengthMax,
-  borderRadius: Tokens.BorderRadiusBase,
-  borderStyle: Tokens.BorderStyleBase,
-  borderWidth: Tokens.BorderWidthBase,
+export const mixins = (
+  tokenOverrides?: TokenOverrideOptions
+): MixinsOptions => {
```

Comment:
I wonder how this change will affect TS types. I know we're overriding `ThemeOptions["mixins"]` with a `define`, but not `MixinsOptions`.

Reply:
Though I didn't see any errors I reverted back to ThemeOptions["mixins"] for consistency